src/views/List.jsx

```diff
@@ -1,13 +1,14 @@
 import { useState } from 'react';
 import { ListItem } from '../components';
+import '../index.css';
```

Comment:
There's no need to import this here. It's already imported in `index.jsx` so it can be used in the whole app :smiley:

Reply:
Aha, thank you for your eagle eyes!! :)